tests/README.md

```diff
@@ -74,6 +150,35 @@ Container-based tests:
 - `--docker`: Uses Docker as container engine. Accepts the base image name.
 - `--container`: Uses Podman as container engine. Accepts the base image name.
 
+To use container backends, use the following options on the command line:
+
+- Podman - `--container <base image name>`
+- Docker - `--docker <base image name>`
+
+This is an example to run test scenarios for rule `rule_sshd_disable_kerb_auth`.
+Note that you need to have root privileges for both cases:
+
+Using Podman:
+
+```
+./test_suite.py rule --container ssg_test_suite --datastream ../build/ssg-centos7-ds.xml --xccdf-id scap_org.open-scap_cref_ssg-rhel7-xccdf-1.2.xml rule_sshd_disable_kerb_auth
+```
+
+Using Docker:
+
+```
+./test_suite.py rule --docker ssg_test_suite --datastream ../build/ssg-centos7-ds.xml --xccdf-id scap_org.open-scap_cref_ssg-rhel7-xccdf-1.2.xml rule_sshd_disable_kerb_auth
+```
+
+Also, as containers may get any IP address, a conflict may appear in your local client's `known_hosts` file.
+You might have a version of `oscap-ssh` that doesn't support ssh connection customization at the client-side, so it may be a good idea to disable known hosts checks for all hosts if you are testing on a VM or under a separate user.
+You can do that by putting following lines in your `$HOME/.ssh/config` file:
+
+```
+StrictHostKeyChecking no
+UserKnownHostsFile /dev/null
+```
+
 ### Profile-based testing
```

Comment:
Should we also mention the `ds_unselect_rules.sh` script in profile-based testing?

Reply:
I've added a section.